gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -3,169 +3,159 @@
+                <a href="/vulnerability/{{ entry }}">{{ entry }}</a>{{ ',' if not loop.last }}
                 {% endfor -%}
                 )
-              {% endif -%}
+                {% endif -%}
               </li>
-            {% endfor -%}
+              {% endfor -%}
             </ul>
           </dd>
-        {% endif -%}
-        <dt>Published</dt>
-        <dd>{{ vulnerability.published }}</dd>
-        <dt>Modified</dt>
-        <dd>{{ vulnerability.modified }}</dd>
-        <dt>Details</dt>
-        <dd class="details">
-          {{ vulnerability.details|markdown|safe -}}
-        </dd>
-        <dt>References</dt>
-        <dd>
-          <ul class="links">
-            {% for reference in vulnerability.references -%}
+          {% endif -%}
+          <dt>Published</dt>
+          <dd>{{ vulnerability.published }}</dd>
+          <dt>Modified</dt>
+          <dd>{{ vulnerability.modified }}</dd>
+          <dt>Details</dt>
+          <dd class="details">
+            {{ vulnerability.details|markdown|safe -}}
+          </dd>
+          <dt>References</dt>
+          <dd>
+            <ul class="links">
+              {% for reference in vulnerability.references -%}
               <li><a href="{{ reference['url'] }}">{{ reference['url'] }}</a></li>
-            {% endfor -%}
-          </ul>
-        </dd>
-      </dl>
+              {% endfor -%}
+            </ul>
+          </dd>
+        </dl>
+      </div>
     </div>
   </div>
-</div>
-<div class="vulnerability-packages-container">
-  <h2 class="title">Affected packages</h2>
-  <spicy-sections
+  <div class="vulnerability-packages-container">
+    <h2 class="title">Affected packages</h2>
+    <spicy-sections
       class="vulnerability-packages{% if vulnerability.affected|should_collapse %} force-collapse{% endif %}">
-    {% for affected in vulnerability.affected -%}
+      {% for affected in vulnerability.affected -%}
       <h2 class="package-header">
         <span class="vuln-ecosystem spicy-sections-workaround">{{ affected['package']['ecosystem'] or 'Git' }}</span>
         <span class="vuln-title-divider spicy-sections-workaround">/</span>
         {# TODO(andrewpollock): it's permissible to have more than one repo, so handle this -#}
-        <span class="vuln-name spicy-sections-workaround">{{ affected['package']['name'] or vulnerability.repo | strip_scheme }}</span>
+        <span class="vuln-name spicy-sections-workaround">{{ affected['package']['name'] or vulnerability.repo |
+          strip_scheme }}</span>
       </h2>
       <div class="mdc-layout-grid">
-        <div class="vulnerability-package-subsection mdc-layout-grid__inner">
-          <h3 class="mdc-layout-grid__cell--span-3">
-            Package Info
-          </h3>
-          <div class="mdc-layout-grid__cell--span-9">
-            <dl>
-              <dt>Package Name</dt>
-              <dd>{{ affected['package']['name'] or 'Unknown' }}</dd>
-              <dt>Repo URL</dt>
```

Comment:
Sorry my comment was very unclear. I was proposing to strip the individual "Package Name" and "Repo URL" bits only in the case when they do not exist, instead of writing "Unknown" or trying to force a repo name into a package name.

Reply:
How about in the interests of getting this merged so the release can proceed, I do that in a followup PR?

Reply:
Or feel free to commit exactly what you have in mind if that's sufficiently expedient...

Reply:
SGTM, I'll approve to unblock.